src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -24,6 +24,7 @@
     
 	public static final String IS_NOT_NULL = "IS NOT NULL";
 	public static final String IS_NULL = "IS NULL";
+	private static GKInstance humanSpeciesInst = new GKInstance();
```

Comment:
Instead of having a human species instance set to an empty `GKInstance`, it might be better to remove this variable and its setter method and have a getter method instead that takes a `dba` object and returns a new human species instance based on the `dba` object passed in: `dba.fetchInstance(48887L)`.  You could cache the value, but it is probably better to just return a new species instance since it could vary by the `dba` passed in and to cache by `dba` object seems like overkill.

Reply:
So I've implemented this (there was already a 'getHuman' method actually in QACheckerHelper), but it really causes the tests to slow down. I assume its because it queries the DB for almost every single instance that it is checking. It used to take 1-2 minutes to run all 5 tests, now its taking 8-10 minutes to run them. Are we OK with such a tradeoff in runtime to avoid caching the Human instance?

Reply:
No, I don't think that's a good trade-off. There is no need to keep querying for the same object again and again and again and again.

Reply:
Also: what was the original problem with a cached object for a single species?